bower.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
@jm3 your comment seems to indicate that you might have been able to get this branch to deploy? To sanity check myself I checked out your branch and deployed to a new app and got the error I have seen with other similar setups:

```
remote: -----> Installing bower dependencies
remote: bower                           ENOENT No bower.json present
remote: Error running: bower --allow-root install 2>&1
remote: -----> Failed trying to compile heroku/ember-cli-deploy
remote:  !     Push rejected, failed to compile emberjs app.
remote: 
remote:  !     Push failed
remote: Verifying deploy....
remote: 
remote: !	Push rejected to thawing-eyrie-85869.
```